src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/CohortDomainServiceTests.cs

```diff
@@ -414,7 +443,7 @@ public CohortDomainServiceTestFixture()
                 Provider = new Mock<Provider>();
                 Provider.Setup(x => x.UkPrn).Returns(ProviderId);
                 Provider.Setup(x => x.CreateCohort(It.IsAny<Provider>(), It.IsAny<AccountLegalEntity>(), null,
-                        It.IsAny<DraftApprenticeshipDetails>(), It.IsAny<UserInfo>()))
+                        null, It.IsAny<UserInfo>()))
```

Comment:
Just out of interest can a provider ever create a cohort with an apprentice?

Reply:
no, just removed this.